packages/gallery-indicator/src/GalleryIndicator/GalleryIndicator.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+
+import { LGIDS } from '../constants';
+
+import {
+  getGalleryIndicatorStyles,
+  getIndicatorStyles,
+} from './GalleryIndicator.styles';
+import { GalleryIndicatorProps } from './GalleryIndicator.types';
+
+export const GalleryIndicator = React.forwardRef<
+  HTMLUListElement,
+  GalleryIndicatorProps
+>(
+  (
+    {
+      darkMode: darkModeProp,
+      count,
```

Comment:
nit: I prefer `length` but not a hill I need to die on

Reply:
i'll update this